scripts/deploySidechain.ts

```diff
@@ -262,18 +290,45 @@ export async function deploySidechainSystem(
         deployOptionsWithCallbacks([poolManagerSetOperator]),
     );
     // Not a constant address
-    const boosterOwner = await deployContractWithCreate2<BoosterOwner, BoosterOwner__factory>(
+    const boosterOwner = await deployContractWithCreate2<BoosterOwnerLite, BoosterOwnerLite__factory>(
         hre,
         create2Factory,
-        new BoosterOwner__factory(deployer),
-        "BoosterOwner",
-        [addresses.daoMultisig, poolManager.address, booster.address, stashFactory.address, ZERO_ADDRESS, true],
+        new BoosterOwnerLite__factory(deployer),
+        "BoosterOwnerLite",
+        [multisigs.daoMultisig, poolManager.address, booster.address, stashFactory.address, ZERO_ADDRESS, true],
         deployOptions,
     );
 
+    const contracts = {
+        voterProxy,
+        booster,
+        boosterOwner,
+        factories: {
+            rewardFactory,
+            stashFactory,
+            tokenFactory,
+            proxyFactory,
+        },
+        poolManager,
+        auraOFT,
+        l2Coordinator,
+    };
+
     let tx: ContractTransaction;
+    // Configure L1 ,L2 Communications
+    if (l1Configuration) {
+        const { addresses: l1ExtConfig, canonical } = l1Configuration;
+        await setTrustedRemoteSidechain(canonical, contracts, l1ExtConfig.canonicalChainId, debug, waitForBlocks);
+    }
+
+    tx = await l2Coordinator.initialize(booster.address, extConfig.token);
+    await waitForTx(tx, debug, waitForBlocks);
+
+    // TODO @phijfry confirm this
```

Comment:
@phijfry  transfer ownship to avoid forgetting doing that in the future.

Reply:
lgtm